repository.yaml

```diff
@@ -1,4 +1,6 @@
+digital-service: Central Reference Data Library Cache
+description: This service simulates the responses of DPS for crdl-cache service
```

Comment:
Maybe we should mention the specific DPS APIs (i.e. "SEED and CSRD Reference Data" and "Customs Office Reference Data")